### ISSUE TITLE
Selecting Material Font in XD forces the XD application to shut down.

### ISSUE THREAD

**Issue by N-5** [google/material-design-icons#1597]
XD ver ' 56.1.12.1
ARM64
Creative Cloud 6.6.4.2

Material Font ver ' 4.0.0

**Comment by tphinney** [comment]
Not sure what you mean by “Material Font”? Do you mean “Material Icons” font? Or “Material Symbols” font? Or…?

Is there any other information provided by the app besides it quitting?

If not, have you reported this to the Adobe XD team?

Even if there is a font bug that is causing this, the Adobe XD team would most likely need to run their special internal Debug version of XD to figure out what is going wrong in the interaction between the app and the font, so as to figure out whether it is an XD issue or a font issue, and if a font issue give some clue as to what is going wrong.

Are you aware that XD is in “maintenance mode”? Adobe says they will still fix bugs, but I expect it to get less attention than a product under active development.

**Comment by tphinney** [comment]
I tested this with a recent version of the Symbols variable font and it worked fine. I went out of my way to use a slightly older version of Adobe XD to better match the bug report. The XD app does not actively support variable fonts, but one can still select different weights in the menu. All worked.

<img width="428" alt="Screenshot 2023-11-08 at 10 56 00 AM" src="https://github.com/google/material-design-icons/assets/6548266/7074e7ab-4fb7-4424-85aa-0dfc2730adeb">
